### DISCUSSION TITLE
How can I sort a logo and heading in the HTMLHeader?

### DISCUSSION THREAD

**Discussion by kieranwalsh** [EvotecIT/PSWriteHTML#360] [Q&A]
If I use this code:

```powershell
New-HTML -TitleText 'This is a test' {
    New-HTMLHeader {
        New-HTMLSection  {
            New-HTMLPanel -Width '10%'
            New-HTMLPanel  {
                New-HTMLText -Text 'List of processes.' -FontSize 20 -Color 'Gray' -FontFamily 'Arial' -FontWeight 'bold'
            } -Width '40%'
            New-HTMLPanel  {            
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Height 80px
            } -Width '40%'
            New-HTMLPanel -Width '10%'
        }
    }
} -ShowHTML -Online
```

I'd expect a small, 10% border on the left and right sides, and then my text and logo to be split evenly across the rest. That doesn't happen:

![image](https://user-images.githubusercontent.com/623629/211526208-8f19abc8-8bfe-45e3-b28b-9bd7045d532b.png)

Also, can I align everything so that my text would be centred in line with the centre of the logo instead of being in line with the top of it? This is what I'd like to achieve:

![image](https://user-images.githubusercontent.com/623629/211527506-7340a504-3785-41da-875f-402fa4a38cce.png)

Thanks for any help.

**Comment by dakhama-mehdi** [comment]
Hello, you can use these 

'''
New-HTML -TitleText 'This is a test' {
    New-HTMLHeader {
        New-HTMLSection  {
           
            New-HTMLPanel -Width '10%'
            New-HTMLPanel  {
                New-HTMLText -LineBreak
                New-HTMLText -LineBreak
                New-HTMLText -Text 'List of processes.' -FontSize 20 -Color 'Gray' -FontFamily 'Arial' -FontWeight 'bold'
            } -Width '40%'
            New-HTMLPanel  {            
                New-HTMLImage -Source 'https://evotec.pl/wp-content/uploads/2015/05/Logo-evotec-012.png' -UrlLink 'https://evotec.pl/' -AlternativeText 'My other text' -Height 80px
            } -Width '40%'
            New-HTMLPanel -Width '10%'
        }
    }
} -ShowHTML -Online
'''

**Comment by dakhama-mehdi** [comment]
your text will be alligned on center with picture 

![image](https://user-images.githubusercontent.com/49924401/213433002-3704ee7c-e0be-4e57-ba89-d050468c5599.png)

**Reply by kieranwalsh** [reply]
Thanks, so you're pushing down the text with those line breaks, ok, I get that. It'll kinda work in that specific instance, but if the logo is bigger, or if I have wrapped text it'll be all out of whack again.

Also, the lining up of the items vertically only looks ok in that size screen, if you go full screen the logo is back to being almost in the middle again:

![image](https://user-images.githubusercontent.com/623629/215331729-0d161594-ddf2-470f-baed-8db2a3d8cab9.png)
